src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Folding/RazorUsingsFoldingProvider.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.LanguageServer.Extensions;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Folding;
+
+internal sealed partial class RazorUsingsFoldingProvider : IRazorFoldingRangeProvider
```

Comment:
Nit: not partial

Reply:
Also, I know you're just following the existing practices in this repo, but IMO including "Razor" in every classname is a little unnecessary. I would be fine if this was just `UsingsFoldingRangeProvider` :D

Reply:
+💯

But also, thank you for doing your best to follow the existing practices. We're slowly trying to work away from them. :smile: